casadm/cas_main.c

```diff
@@ -294,6 +294,11 @@ int handle_start()
 	int status;
 	struct stat device_info;
 
+	if (command_args_values.state && command_args_values.force) {
```

Comment:
Please explicitly compare state with define value for better readability of the code:
`command_args_values.state == CACHE_INIT_LOAD`